api/cloudcontroller/ccv3/paginate.go

```diff
@@ -2,10 +2,15 @@ package ccv3
 
 import (
 	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
 
 	"code.cloudfoundry.org/cli/api/cloudcontroller"
 )
 
+const MaxResultsPerPage int = 5000
```

Comment:
**suggestion** Use the existing definition here https://github.com/cloudfoundry/cli/blob/6bed63ee311dd978cd2bf343c10b47601f45a2e1/api/cloudcontroller/ccv3/query.go#L129